sql_generators/glean_usage/templates/metrics_clients_last_seen.query.sql

```diff
@@ -1,3 +1,25 @@
+{{ header }}
+
+{% raw %}{% if is_init() %}{% endraw %}
+SELECT
+  DATE(@submission_date) AS submission_date,
```

Comment:
I don't think using the `@submission_date` parameter in the init use case makes sense.
```suggestion
  submission_date,
```